__tests__/app.test.js

```diff
@@ -87,19 +87,43 @@ describe('GET /api/reviews/:review_id', () => {
         return request(app)
         .get(`/api/reviews/${reviewId}`)
         .expect(400)
-        .then(({body})=>{
+        .then(({body}) => {
         expect(body).toEqual({msg: 'path not found!'});
         });
     });
-    });
-
-    describe('GET incorrect review path',()=>{
-        test('404: non-existent route', ()=>{
+        test('404: non-existent route', () =>{
             return request(app)
             .get('/api/reviews/66777')
             .expect(404)
             .then(({body}) => {
             expect(body).toEqual({msg: 'Not Found'});
             });
         });
-    });
\ No newline at end of file
+    });
+
+    describe('GET /api/users', () => {
+        test('200: responds with an array of category objects, each of which should have the appropriate properties', () => {
+            return request(app)
+            .get('/api/users').expect(200)
+            .then((response) => {
+                const {users} = response.body;
+            users.forEach((user) => {
+            expect(user).toMatchObject({
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String),
+            });
+            });
+        });
+        });
+        test('404: non-existent route', ()=>{
+            return request(app)
+            .get('/apz')
+            .expect(404)
+            .then(({body})=>{
+            expect(body).toEqual({msg: 'path not found!'});
+            });
```

Comment:
I would put this in it's own describe block because it doesn't really link to any endpoint. We also can just check this once!